### ISSUE TITLE
New installation can't find flake8_polyfill library.

### ISSUE THREAD

**Issue by proofit404** [rubik/xenon#38]
```
certifi==2020.6.20,chardet==3.0.4,colorama==0.4.3,future==0.18.2,idna==2.10,mando==0.6.4,PyYAML==5.3.1,radon==4.3.1,requests==2.24.0,six==1.15.0,urllib3==1.25.10,xenon==0.7.0
```

```
Traceback (most recent call last):
  File "/home/proofit404/data/generics/.tox/xenon/bin/xenon", line 8, in <module>
    sys.exit(main())
  File "/home/proofit404/data/generics/.tox/xenon/lib/python3.8/site-packages/xenon/__init__.py", line 72, in main
    from xenon.core import analyze
  File "/home/proofit404/data/generics/.tox/xenon/lib/python3.8/site-packages/xenon/core.py", line 5, in <module>
    from radon.complexity import cc_rank, SCORE
  File "/home/proofit404/data/generics/.tox/xenon/lib/python3.8/site-packages/radon/complexity.py", line 7, in <module>
    from flake8_polyfill import options
ModuleNotFoundError: No module named 'flake8_polyfill'
```

Maybe its a `radon` issue, but I've got the error when installing `xenon` tool.

**Comment by alanverdugo** [comment]
Hello, in case it helps:

As mentioned above, this seems to be related to the newest radon's release, which was done some hours ago:
https://pypi.org/project/radon/#history

`flake8-polyfill` was moved from `install_requires` to `extras_require` in radon's `setup.py`
Related commits:
https://github.com/rubik/radon/commit/2c9627de5fccf7bb878283c340c61c0cd5042db8
https://github.com/rubik/radon/commit/2a8673cf5926c0e9c146dcde128812fa01cf11fa

**Comment by proofit404** [comment]
That make sense, but this error happens even that I didn't run it as a `flake8` part.

```
xenon --max-absolute A --max-modules A --max-average A src/
```

**Comment by rubik** [comment]
@proofit404 @alanverdugo Thanks, this was an oversight on my part. I updated Xenon requirements and released version 0.7.1 to PyPI. I have verified that the issue is no longer present in a fresh installation.
